lib/u3d/installer.rb

```diff
@@ -63,7 +63,8 @@ def self.sanitize_installs(installer)
     def self.install_modules(files, version, installation_path: nil)
       installer = Installer.create
       files.each do |name, file, info|
-        UI.verbose "Installing #{name}#{info['mandatory'] ? ' (mandatory package)' : ''}, with file #{file}"
```

Comment:
We could use `info['title']` as well somewhere.

Note: do this make the console much more verbose?

Reply:
Regarding verbose, this adds a few extra lines, but I really feel like the added cost is overweight by the increase readability, ie
__Before:__
```
$ C:/Users/pniez/Downloads/Unity_Packages/2018.1.0b2/UnitySetup64.exe -UI=reduced /S /D=C:\Program Files\Unity_2018.1.0b2
No pty implementation found. Falling back to popen. Output might be buffered
Successfully installed Unity 2018.1.0b2
```
__After:__
```
--------------------------------
--- Installing Documentation ---
--------------------------------
Installing with C:/Users/pniez/Downloads/Unity_Packages/2017.1.2f1/UnityDocumentationSetup.exe
$ C:/Users/pniez/Downloads/Unity_Packages/2017.1.2f1/UnityDocumentationSetup.exe /S /D=C:\Program Files\Unity_2017.1.2f1
No pty implementation found. Falling back to popen. Output might be buffered
Successfully installed Documentation
```
WDYT?

Reply:
OK. Any use of `info['title']`?

Reply:
Already implemented :)